blueocean/src/test/java/io/jenkins/blueocean/config/JenkinsJSExtensionsTest.java

```diff
@@ -55,7 +55,7 @@ public void test() {
             Assert.assertNotNull(extensionPoints);
 
             if ("blueocean-dashboard".equals(pluginId)) {
-                Assert.assertEquals(7, extensionPoints.size());
+                Assert.assertEquals(9, extensionPoints.size());
```

Comment:
That is in the age of times. There are even notes about it https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/js/jenkins-js-extension.yaml#L3